stream.js

```diff
@@ -6,7 +6,10 @@ module.exports = WebSocketStream
 
 function WebSocketStream(target, protocols, options) {
   var stream, socket
-  var socketWrite = process.title === 'browser' ? socketWriteBrowser : socketWriteNode
+
+  var isBrowser = process.title === 'browser'
+  var isNative = global.WebSocket
```

Comment:
IMHO *is*Something variables should be booleans, maybe add a !! before

On a side note, why isn't the check for isBrowser enough?